product-service/handlers/getProductsById/getProductsById.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { withCorsHeaders } from "../../utils/withCorsHeaders";
+import { getProductsById } from ".";
+
+const mockProductList = [
+  {
+    count: 1,
+    description: "Description",
+    id: "1",
+    price: 10,
+    title: "Toy",
+    src: "url",
+  },
+];
+
+jest.requireActual("node-fetch");
+jest.mock("../../data/productList.json", () => [
```

Comment:
It can be 
```js
jest.mock("../../data/productList.json", () => mockProductList)
```

Reply:
No, there will be an error
`ReferenceError: Cannot access 'mockProductList' before initialization`

Reply:
@svetlanagorchanina oh! This is because of **jest** hoisting mocks. Nevermind)

Reply:
Right, i didn't find any better solution, so any suggestions are welcome

Reply:
@svetlanagorchanina I have got only one solution. But it's pretty weird:
```js
const mockProductList = [
  {
    count: 1,
    description: "Description",
    id: "1",
    price: 10,
    title: "Toy",
    src: "url",
  },
];

import { withCorsHeaders } from "../../utils/withCorsHeaders";
import { getProductsById } from ".";

jest.mock("../../data/productList.json", () => mockProductList)
...
```

Reply:
I've tried it and it didn't work either : (